tests/test_coordUtils.py

```diff
@@ -266,8 +273,9 @@ def testComputeAzAltFromPupilBaseWithVectorPupil100(self):
                 if sep.asRadians() > 1e-14:
                     print("Warning: sep={:0.5f} asec for vectorPupilScaled={}, vectorBaseScaled={}".format(
                           sep.asArcseconds(), vectorPupilScaled, vectorBaseScaled))
-                # the worst error I see is 0.0026" for vectorBase=(1, 0.7, -0.8)
-                # that's bad enough to scary, but is acceptable
+                # The worst error I see is 0.0026"
+                # for vectorBase=(1, 0.7, -0.8)
+                # that's bad enough to scary, but is acceptable.
```

Comment:
"that's bad enough to scary"

Reply:
I reworded it to "That is worrisome, but acceptable."